.gitignore

```diff
@@ -87,3 +87,13 @@ tools/fog-local-network/*_pb2*.py
 
 # libmobilecoin/rust-bitcode build
 libmobilecoin/rust-bitcode/*
+
+# build automation tmp
+.tmp
+**/charts/*.tgz
+**/Chart.lock
```

Comment:
Optional/Equivalent: `**/` is the default/implicit prefix
```suggestion
charts/*.tgz
Chart.lock
```

Reply:
Again it works as intended right now. Maybe a follow up to test and confirm